bin/typescript-fetch-petstore-typescript-three-plus.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
Please add a windows-version in /bin/windows, too

Reply:
Maybe this script still needs to be made executable, i.e.

chmod u+x bin/typescript-fetch-petetore-typescript-three-plus.sh

Reply:
The command was `git update-index --chmod=+x bin/typescript-fetch-petstore-typescript-three-plus.sh` for me (using WSL/Windows so maybe that's what screwed this up)